VERSION

```diff
@@ -3,9 +3,9 @@
 # Copyright (c) 2011      NVIDIA Corporation.  All rights reserved.
 # Copyright (c) 2013      Mellanox Technologies, Inc.
 #                         All rights reserved.
-# Copyright (c) 2017      Los Alamos National Security, LLC.  All rights
+# Copyright (c) 2017-2019 Los Alamos National Security, LLC.  All rights
```

Comment:
@gpaulsen if you want to update VERSION for Los alamos, we have a new company operating the lab.  git grep for triad in some other files and you'll see the right attribution.  Or I can fix this.

Reply:
i can fix.